porcupine/plugins/filemanager.py

```diff
@@ -221,6 +281,8 @@ def is_NOT_project_root(path: Path) -> bool:
     # Doing something to an entire project is more difficult than you would think.
     # For example, if the project is renamed, venv locations don't update.
     # TODO: update venv locations when the venv is renamed
+    Command("Copy", "<<Copy>>", (lambda p: not p.is_dir()), copy),
+    Command("Paste here", "<<Paste>>", (lambda p: p.is_dir() and copy_path is not None), paste),
```

Comment:
After trying this out, I found "Paste here" to be quite confusing. If I have a file selected and I press `Ctrl+C Ctrl+V`, I would expect to get a copy of the file in the same directory where it is. Maybe we should have two options, "Paste" (possible whenever the selection is not a project, I would prefer this to be Ctrl+V) and "Paste inside selected directory" (the current "Paste here")?

Reply:
Hi @Akuli, I think you suggested the floowing behaviour:
(Ctrl + V) + no dir selected -> copy file inside same directory
(Ctrl + V or (right-click + "Paste inside selected directory)) + dir selected -> copy file inside selected directory

P.S.
What to do in case: (Ctrl + V) + file selected? (I think that if the selected file is in another directory and we press Ctrl + V and paste in the same directory as in the first rule it would be a little confusing... Should we block Ctrl + V or should we let the copy be pasted in the directory which contains the selected file (this is how IntelliJ works) ? )

Reply:
If I have `foo/bar.py` selected and I press Ctrl+V, it will copy to `foo/blah.py`.

If I have `foo/subfolder/` selected and I press Ctrl+V, that too will copy to `foo/blah.py`.

If I have `foo/subfolder/` selected and I select "Paste inside selected directory", it will copy to `foo/subfolder/blah.py`.

So it does not matter what you have selected, what matters is where the selected file or directory is. In all cases, the new "Paste here" would do same as Ctrl+V. If you have a better suggestion to use instead of "Paste here", feel free to name it better.